From bdc3b4ebac0f4e7bd85a0af9cae80f70df336ba7 Mon Sep 17 00:00:00 2001
From: Namit Jain <namit@apache.org>
Date: Tue, 12 Jun 2012 23:12:03 +0000
Subject: [PATCH 087/148] HIVE-3081 ROFL Moment. Numberator and denaminator typos
 (Edward Capriolo via namit)

git-svn-id: https://svn.apache.org/repos/asf/hive/trunk@1349584 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit cbdc10f13f4ba83d42742a86f5b30cbd29091fb8)
---
 .../java/org/apache/hadoop/hive/ql/ErrorMsg.java   |    4 ++--
 .../hadoop/hive/ql/parse/SemanticAnalyzer.java     |    2 +-
 ql/src/test/results/clientnegative/sample.q.out    |    2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/ql/src/java/org/apache/hadoop/hive/ql/ErrorMsg.java b/src/ql/src/java/org/apache/hadoop/hive/ql/ErrorMsg.java
index f7032b0..73575e5 100644
--- a/src/ql/src/java/org/apache/hadoop/hive/ql/ErrorMsg.java
+++ b/src/ql/src/java/org/apache/hadoop/hive/ql/ErrorMsg.java
@@ -114,8 +114,8 @@ public enum ErrorMsg {
   INVALID_MAPJOIN_HINT(10058, "Neither table specified as map-table"),
   INVALID_MAPJOIN_TABLE(10059, "Result of a union cannot be a map table"),
   NON_BUCKETED_TABLE(10060, "Sampling expression needed for non-bucketed table"),
-  BUCKETED_NUMBERATOR_BIGGER_DENOMINATOR(10061, "Numberator should not be bigger than "
-      + "denaminator in sample clause for table"),
+  BUCKETED_NUMERATOR_BIGGER_DENOMINATOR(10061, "Numerator should not be bigger than "
+      + "denominator in sample clause for table"),
   NEED_PARTITION_ERROR(10062, "Need to specify partition columns because the destination "
       + "table is partitioned"),
   CTAS_CTLT_COEXISTENCE(10063, "Create table command does not allow LIKE and AS-SELECT in "
diff --git a/src/ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java b/src/ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java
index 1086a8d..519573b 100644
--- a/src/ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java
+++ b/src/ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java
@@ -6532,7 +6532,7 @@ public class SemanticAnalyzer extends BaseSemanticAnalyzer {
 
       if (num > den) {
         throw new SemanticException(
-            ErrorMsg.BUCKETED_NUMBERATOR_BIGGER_DENOMINATOR.getMsg() + " "
+            ErrorMsg.BUCKETED_NUMERATOR_BIGGER_DENOMINATOR.getMsg() + " "
             + tab.getTableName());
       }
 
diff --git a/src/ql/src/test/results/clientnegative/sample.q.out b/src/ql/src/test/results/clientnegative/sample.q.out
index 001440d..3f70f16 100644
--- a/src/ql/src/test/results/clientnegative/sample.q.out
+++ b/src/ql/src/test/results/clientnegative/sample.q.out
@@ -1 +1 @@
-FAILED: SemanticException [Error 10061]: Numberator should not be bigger than denaminator in sample clause for table srcbucket
+FAILED: SemanticException [Error 10061]: Numerator should not be bigger than denominator in sample clause for table srcbucket
-- 
1.7.0.4

